types/styled-components/test-utils.d.ts

```diff
@@ -0,0 +1,7 @@
+import type { StyledComponent } from './index';
+
+export function find(element: HTMLElement, styledComponent: StyledComponent<any, any, any, any>): HTMLElement | null;
```

Comment:
Can you please confirm that this is intentionally `null`, rather than `undefined`?

Reply:
Seems fine to me since this is the documented value. If there's a difference at runtime, then we should raise this against documentation first.

Reply:
Could also dig into source and use the return type of whatever Dom method is used internally.